recruiter/tests.py

```diff
@@ -46,20 +83,58 @@ def test_get_all_recruiters_of_a_specified_company_returns_all_recruiters_as_a_q
 
 @pytest.mark.django_db
 def test_if_user_is_a_recruiter_returns_true_if_the_auth_user_is_associated_with_a_recruiter_account(example_recruiter):
-    example_user_not_recruiter = User.objects.create_user("notrecruiter", "password")
+    example_user_not_recruiter = User.objects.create_user(
+        "notrecruiter", "password")
     example_user_not_recruiter.save()
     assert Recruiter.is_recruiter(example_user_not_recruiter.id) is False
     assert Recruiter.is_recruiter(example_recruiter.user.id) is True
 
 
 @pytest.mark.django_db
 def test_get_user_as_recruiter_object(example_recruiter):
-    assert isinstance(Recruiter.get_recruiter(example_recruiter.user.id), Recruiter)
+    assert isinstance(Recruiter.get_recruiter(
+        example_recruiter.user.id), Recruiter)
 
 
 @pytest.mark.django_db
 def test_get_recruiter_raises_exception_for_non_recruiter_user():
-    example_user_not_recruiter = User.objects.create_user("notrecruiter", "password")
+    example_user_not_recruiter = User.objects.create_user(
+        "notrecruiter", "password")
     example_user_not_recruiter.save()
     with pytest.raises(Recruiter.DoesNotExist):
         Recruiter.get_recruiter(example_user_not_recruiter.id)
+
+
+@pytest.mark.django_db
+def test_getting_the_correct_url(rf, admin_user):
```

Comment:
Why use `RequestFactory` here and not `client`?

Reply:
I saw that in the menu that you sent to the guy a few days ago. but I will change that.